api/src/pcapi/routes/backoffice/templates/chronicles/details.html

```diff
@@ -0,0 +1,194 @@
+{% from "components/presentation/details/tabs.html" import build_details_tab %}
+{% from "components/presentation/details/tabs.html" import build_details_tabs_wrapper %}
+{% from "components/presentation/details/tabs.html" import build_details_tab_content %}
+{% from "components/presentation/details/tabs.html" import build_details_tabs_content_wrapper %}
+{% from "components/history/view.html" import build_history_view with context %}
+{% from "components/generic_modal.html" import build_modal_form with context %}
+{% extends "layouts/standard.html" %}
+{% block main_content %}
+  <div class="row row-cols-1 g-4 py-3">
+    <div class="col">
+      <div class="card shadow">
+        <div class="card-body">
+          <div class="row">
+            <div class="d-flex justify-content-between">
+              <div class="p-2">
+                {% if chronicle.firstName %}
+                  <h2 class="card-title mb-3 text-primary">
+                    Chronique de {{ chronicle.firstName }}
+                    {% if chronicle.age %}, {{ chronicle.age }} ans{% endif %}
+                  </h2>
+                {% else %}
+                  <h2 class="card-title mb-3 text-primary">{{ chronicle.id }}</h2>
+                {% endif %}
+                <p class="card-subtitle text-muted mb-3 h5">Chronicle ID : {{ chronicle.id }}</p>
+              </div>
+              <h5 class="p-2">
+                {% if chronicle.isActive %}
+                  <span class="badge rounded-pill text-bg-success align-middle">Publié</span>
+                {% else %}
+                  <span class="badge rounded-pill text-bg-warning align-middle">Non publié</span>
```

Comment:
Si on parle d'une chronique, c'est au féminin : 
```suggestion
                  <span class="badge rounded-pill text-bg-success align-middle">Publiée</span>
                {% else %}
                  <span class="badge rounded-pill text-bg-warning align-middle">Non publiée</span>
```